### ISSUE TITLE
CHAPTER: Weights

### ISSUE THREAD

**Issue by darribas** [gdsbook/book#26]
Thread to discuss chapter on Spatial Weights. It picks discussion up from [gds#40](https://github.com/sjsrey/gds/issues/40).

**Comment by darribas** [comment]
Final pass:

- [ ] Dani
- [ ] Serge
- [ ] Levi

**Comment by darribas** [comment]
Dani final pass tracker, to be edited as we go along:

Things to ensure (from #152 ):

- [ ] Library imports (first time inline, afterwards at the top of the chapter
- [ ] No aliases on imports (`import pandas`)
- [ ] Ensure no `XXX` for citations
- [ ] Ensure `matplotlib` RC file is picked up and figures are standard
- [ ] Remove `%matplotlib inline`
- [ ] Graphics
     - No style
     - No title (unless multi-axis titles, but not `suptitle`)
     - No axes for maps (`ax.set_axis_off()`)
- [ ] Code comments are fine if needed
- [ ] Hide/remove code cells that we want to hide (see docs [here](https://jupyterbook.org/interactive/hiding.html))
- [ ] Typesetting check
- [ ] General flow of the text
- [ ] Ensure there is a "next step" section at the end of the chapter pointing to further references for the topic (re. #137)

**Comment by darribas** [comment]
Notes from my review:

- I've streamlined the visualisation section because I though there was too much repetition and not enough content (I've added KNN to the figure for completion)
- My sense is that the use case with boundaries is well above the level of the book, both conceptually and technically. There are snippets I struggle to read through... But I think it's interesting enough it's worth leaving in. I've added a sentence at the beginning signposting this so readers don't freak out.
